backend/routes/users.js

```diff
@@ -112,6 +112,9 @@ app.get("/currentUser", (req, res, next) => {
 app.get("", (req, res, next) => {
     User.find(
         ).then(userResult => {
+	for(var i = 0; i < userResult.length; i++) {
```

Comment:
userResult.forEach( user => user.password = "<censored>") or something like this

Reply:
its stupid but lets do it right